src/OrchardCore.Modules/OrchardCore.Spatial/Views/GeoPointField-Leaflet.Edit.cshtml

```diff
@@ -27,5 +27,5 @@
         <div id="map" style="height : 200px"></div>
     </div>
 </div>
-<script asp-name="coordinate-leaflet-field" depends-on="leaflet,jQuery" asp-src="~/OrchardCore.Spatial/Scripts/coordinate-leaflet-field.js" debug-src="~/OrchardCore.Spatial/Scripts/coordinate-leaflet-field.js" type="text/javascript" at="Foot"></script>
-<style asp-name="leaflet" version="1.3.1" asp-src="~/OrchardCore.Spatial/leaflet/leaflet.css"></style>
\ No newline at end of file
+<script asp-name="coordinate-leaflet-field" depends-on="leaflet,jQuery" asp-src="~/OrchardCore.Spatial/Scripts/coordinate-leaflet-field.min.js" debug-src="~/OrchardCore.Spatial/Scripts/coordinate-leaflet-field.js" type="text/javascript" at="Foot"></script>
+<style asp-name="leaflet" version="1.7.1" asp-src="~/OrchardCore.Spatial/Styles/leaflet.css"></style>
```

Comment:
`version="1"` could be enough.

Reply:
Well it reflects it's true version

Reply:
I think I will keep it like it is unless you say we did this everywhere else?

Reply:
We never use both `asp-src` and `version` at the same time. 
It will use the `asp-name` to get the style defined in ResourceOptions.

Reply:
True both of these just need asp-name

Reply:
Well nevermind you are right. Because the script doesn't have a CND I shouldn't register it in the ResourceManifest